lib/nerves_firmware_ssh/application.ex

```diff
@@ -26,7 +29,7 @@ defmodule Nerves.Firmware.SSH.Application do
         {:max_sessions, 1},
         {:id_string, :random},
         {:key_cb, {Nerves.Firmware.SSH.Keys, cb_opts}},
-        {:system_dir, :code.priv_dir(:nerves_firmware_ssh)}, # FIXME
+        {:system_dir, to_char_list(system_dir)},
```

Comment:
While I did put `#FIXME` here, I wondering if hardcoding the server keys in the priv dir is much worse than making the user hardcode them in `/etc/ssh`. Given that the server's private ssh key is in the public unless the user goes through even more work to generate them on device, I've been thinking that we might as well make this case really easy. How about this as a strategy:
1. If the user specifies a `:system_dir` in their config, use it.
2. If the `/etc/ssh` directory exists, use it.
3. Otherwise, use `:nerves_firmware_ssh`'s keys in its priv dir.